etc/build/Dockerfile.tpl

```diff
@@ -90,6 +94,9 @@ WORKDIR $DRIVER_REPO_PATH/
 
 ENV GO111MODULE=on GOFLAGS=-mod=vendor
 
+# workaround for https://github.com/golang/go/issues/28065
```

Comment:
Why is this one not also conditional, as in Stage 1.1 above?

Reply:
The previous workaround is inside the "native test" stage, so it applies only when the native language is Go. In this case, the stage is a "server build" which is always in Go.